src/ui/components/VPNSystemAlert.qml

```diff
@@ -107,6 +104,20 @@ VPNAlert {
                 visible: true
             }
 
+        },
+        State {
+            name: VPN.GeoIpRestrictionAlert
+
+            PropertyChanges {
+                target: alertBox
+                alertType: "geoip-restriction"
+                //% "Operation not available where you are located"
```

Comment:
I can't figure out what this means, we'll need a localization comment.

Can you explain to me, possibly rephrasing it, what "located" means in this context? When is a user located?